timesketch/models/sketch.py

```diff
@@ -143,6 +145,20 @@ def active_timelines(self):
             _timelines.append(timeline)
         return _timelines
 
+    def get_active_analysis_sessions(self):
+        """List active analysis sessions.
+
+        Returns:
+            List of instances of timesketch.models.sketch.AnalysisSession
+        """
+        active_sessions = []
+        for session in self.analysissessions:
+            for analysis in session.analyses:
+                if analysis.get_status.status in ('PENDING', 'STARTED'):
+                    active_sessions.append(session)
+                    break
```

Comment:
why the break here?

Reply:
I break the inner loop on first hit of an active analysis in the session as this is enough for the session to be marked as active. Saves a couple or rounds.

Reply:
Added a comment

Reply:
I read the code again... I thought this was `completed`, but of course this is just that if one analyzer is still in a a pending/started stage then it is active, which is of course right, so yes... this is fine